RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,157 @@
+#ifndef Constants_cuh
+#define Constants_cuh
+
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+#ifdef CACHE_ALLOC
+#include "HeterogeneousCore/AlpakaInterface/interface/CachedBufAlloc.h"
+#endif
+
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
+#include <cuda_fp16.h>
+#endif
+
+namespace SDL {
```

Comment:
I read in the [CMSSW coding rules](https://cms-sw.github.io/cms_coding_rules.html) that namespaces should use lowercase.
@slava77 Is this acceptable or should I start changing this preemptively?

Reply:
I would not change now